src/tests/multihost/alltests/test_proxy_rfc2307.py

```diff
@@ -0,0 +1,625 @@
+        """
+        config_sssd(multihost)
+        tools = sssdTools(multihost.client[0])
+        sssd_params = {'domains': ds_instance_name}
+        tools.sssd_conf('sssd', sssd_params)
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'use_fully_qualified_names': 'true'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        for command in ["getent passwd User_CS1@example1",
+                        "getent group User_CS1_grp1@example1",
+                        "id User_CS1@example1"]:
+            execute_cmd(multihost, command)
+        client_hostip = multihost.client[0].ip
+        client = pexpect_ssh(client_hostip, "User_CS1@example1", 'Secret123', debug=False)
+        try:
+            client.login(login_timeout=30, sync_multiplier=5,
+                         auto_prompt_reset=False)
+        except SSHLoginException:
+            pytest.fail("%s failed to login" % "User_CS1@example1")
+        else:
+            client.logout()
+
+    def test_min_id_max_id(self, multihost, backupsssdconf):
+        """
+        :title: min id max id
+        :id: 02ed13ec-4333-11ed-86e7-845cf3eff344
+        :caseposneg: positive
+        :steps:
+          1. Tests for min id max id
+        :expectedresults:
+          1. Should succeed
+        """
+        config_sssd(multihost)
+        tools = sssdTools(multihost.client[0])
+        sssd_params = {'domains': ds_instance_name}
+        tools.sssd_conf('sssd', sssd_params)
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'min_id': '111112'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        for command in ["getent passwd User_CS1",
+                        "getent group User_CS1_grp1"]:
+            with pytest.raises(subprocess.CalledProcessError):
+                execute_cmd(multihost, command)
+        execute_cmd(multihost, "sed -i '/min_id/d' /etc/sssd/sssd.conf")
+        execute_cmd(multihost, 'echo "max_id=111110" >> /etc/sssd/sssd.conf')
+        tools.clear_sssd_cache()
+        for command in ["getent passwd User_CS1",
+                        "getent group User_CS1_grp1"]:
+            with pytest.raises(subprocess.CalledProcessError):
+                execute_cmd(multihost, command)
+        execute_cmd(multihost, "sed -i '/max_id/d' /etc/sssd/sssd.conf")
+        execute_cmd(multihost, 'echo "min_id=111110" >> /etc/sssd/sssd.conf')
+        execute_cmd(multihost, 'echo "max_id=111112" >> /etc/sssd/sssd.conf')
+        tools.clear_sssd_cache()
+        for command in ["getent passwd User_CS1",
+                        "getent group User_CS1_grp1",
+                        "id User_CS1"]:
+            execute_cmd(multihost, command)
+
+    def test_case_sensitive_false_lookup_user_group(self, multihost, backupsssdconf):
+        """
+        :title: case sensitive is false simple deny users is user cs1
```

Comment:
title is still having user cs1 is denied

Reply:
Done